bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -77,8 +77,8 @@ <h2 class="addendum-subtitle">{{ ftl('manifesto-pledge-for-a-healthy-internet')
       include_cta=True,
     ) %}
       <p>
-        <a class="mzp-c-button button-manifesto js-manifesto-share" target="_blank" rel="noopener noreferrer" href="{{ twitter_link('custom', ftl('manifesto-i-support-the-vision-of')) }}" data-link-type="social share" data-link-text="Twitter">
-          {{ ftl('manifesto-share-on-twitter') }}
+        <a class="mzp-c-button button-manifesto js-manifesto-share" target="_blank" rel="noopener noreferrer" href="{{ twitter_link('custom', ftl('manifesto-i-support-the-vision-of')) }}" data-link-type="social share" data-link-text="X (formerly Twitter)" aria-label='{{ ftl('manifesto-share-on-x-aria-label') }}'>
```

Comment:
Again, I think we're maybe over-using `aria-label` here. If we don't think the existing link text gives enough context already, then it's a problem not just for screen readers? I get that we want to keep the CTA short, but I think "Share on X" is probably fine?

Reply:
Yeah, I mentioned it in a comment above as well, but I just don't want screenreader users to get confused with "X" because it can mean a myriad of other things. I can add `formerly Twitter` to it to make it clearer, but I wonder if that's visual overuse for folks who don't use screenreaders?

Reply:
Ok, I think that makes sense, let's leave this one as-is 👍